lib/usbd/usbd_ep0.c

```diff
@@ -208,7 +208,7 @@ static int utf8_to_utf16(const uint8_t *utf8, uint16_t *utf16,
 
 			/* trailing bytes payload */
 			while (trailing_bytes_utf8-- > 0) {
-				utf32_ch = (utf32_ch << 6) | (utf8[i++] & 0b00111111);
+				utf32_ch = (utf32_ch << 6) | (utf8[i++] & 0x3f);
 			}
```

Comment:
there was a specific reason (readability imo) to write these specific constant in binary notation.
but hexadecimal notation wont do any improvement either.